gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -256,4 +261,88 @@ public URL signUrl(long expirationTimeInSeconds, SignUrlOption... options) {
   public Storage storage() {
     return storage;
   }
+
+  /**
+   * Gets the requested blobs. If {@code infos.length == 0} an empty list is returned. If
```

Comment:
To me the empty case sounds obvious (and I don't think we stated it in other places) but fine as is.

Actually, what I think is a problem is that in some cases we are returning immutable list (for empty or singleton)
and in others we don't.